apps/helpers/grgsm_capture

```diff
@@ -139,11 +139,11 @@ class grgsm_capture(gr.top_block):
         if self.verbose or self.burst_file:
             self.gsm_receiver.set_cell_allocation([self.arfcn])
             if options.band:
-                new_freq = grgsm.arfcn.arfcn2downlink(self.arfcn, self.band)
+                new_freq = grgsm.arfcn.arfcn2downlink(self.arfcn)
             else:
                 for band in grgsm.arfcn.get_bands():
-                    if grgsm.arfcn.is_valid_arfcn(arfcn, band):
-                        new_freq = grgsm.arfcn.arfcn2downlink(arfcn, band)
+                    if grgsm.arfcn.is_valid_arfcn(arfcn):
+                        new_freq = grgsm.arfcn.arfcn2downlink(arfcn)
```

Comment:
Do you still need the for loop when "band" is no longer needed?

Reply:
No,there is no need for the for loop.^_^
Yesterday,I was just trying to make it work,didn't think it through. 
Now i'm kind of thinking there is no need for option:'band' if option:'arfcn' is set.